tests/test_run.py

```diff
@@ -0,0 +1,244 @@
+import copy
+import subprocess
+
+import pytest
+
+from little_cheesemonger._errors import LittleCheesemongerError
+from little_cheesemonger._run import (
+    execute_steps,
+    install_python_dependencies,
+    install_system_dependencies,
+    run,
+    run_subprocess,
+    set_environment_variables,
+)
+from tests.constants import ARCHITECTURE, PLATFORM, PYTHON_BINARIES, PYTHON_VERSION
+
+ENVIRONMENT_VARIABLES = ["foo=bar"]
+SYSTEM_DEPENDENCIES = ["foo-1.0.0"]
+PYTHON_DEPENDENCIES = ["foo==1.0.0"]
+STEPS = ["foo"]
+CONFIGURATION = {
+    "environment_variables": ENVIRONMENT_VARIABLES,
+    "system_dependencies": SYSTEM_DEPENDENCIES,
+    "python_dependencies": PYTHON_DEPENDENCIES,
+    "steps": STEPS,
+}
+
+
+@pytest.fixture
+def set_environment_variables_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.set_environment_variables")
+
+
+@pytest.fixture
+def install_system_dependencies_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.install_system_dependencies")
+
+
+@pytest.fixture
+def install_python_dependencies_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.install_python_dependencies")
+
+
+@pytest.fixture
+def execute_steps_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.execute_steps")
+
+
+@pytest.fixture
+def run_subprocess_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.run_subprocess")
+
+
+@pytest.fixture
+def get_python_binaries(mocker):
+    return mocker.patch(
+        "little_cheesemonger._run.get_python_binaries",
+        return_value=PYTHON_BINARIES[ARCHITECTURE][PLATFORM],
+    )
+
+
+@pytest.fixture
+def os_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.os")
+
+
+@pytest.fixture
+def subprocess_run_mock(mocker):
+    return mocker.patch("little_cheesemonger._run.subprocess.run")
```

Comment:
```suggestion
    return mocker.patch("subprocess.run")
```
Since this is a stdlib module, it removing the chseemonger prefix makes it clearer what is happening.